src/XML/Operator.php

```diff
@@ -73,7 +73,9 @@ static function xmlDeserialize(Reader $reader): Operator {
 					$operator->arguments[] = $argument['value'];
 				}
 			} else {
-				$reader->read();
+			       if (!$reader->read()) {
```

Comment:
Do we still need this?

Reply:
Theoretically we don't need this, however,  I strongly recommend this check the return to avoid these kinds of  infinite loops  because incoming XML requests are very wild and sometimes parser may behave unexpectedly for such requests.